snp_application/include/snp_widget.h

```diff
@@ -21,9 +21,11 @@ class SNPWidget : public QWidget
 public:
   explicit SNPWidget(rclcpp::Node::SharedPtr rviz_node, QWidget* parent = nullptr);
 
-private:
+protected:
   void runTreeWithThread();
```

Comment:
Should `runTreeWithThread()` still be protected?

Reply:
I think so, just in case someone wants to override it for some reason